### ISSUE TITLE
Disable audience verification if audience not configured

### ISSUE THREAD

**Issue by mianelson** [dmi3y/flask-jwt-consumer#1]
Even if `config.audience` (via `JWT_IDENTITY`) is not  configured for a service, a token which includes `'aud'` in JWT claims will automatically be verified. This means we cannot disable audience checking by passing `audience=None` alone. We should also set the `verify_aud` option to `False` if we have not configured `JWT_IDENTITY` for the service consuming JWTs. Example below in decorators.py (I did not have contributor permissions to submit a PR so I've copied the fix below): 
```
def requires_jwt(f):
    """Determines if the Access Token is valid."""
    @wraps(f)
    def decorated(*args, **kwargs):
        token = get_jwt_raw()
        key = _brute_force_key(token)
        if key:
            try:
                jwt_config = {
                    'algorithms': config.algorithm
                }
                if config.audience:
                    jwt_config.update({
                        'audience': config.audience
                    })
                else:
                    jwt_config.update({'options': {'verify_aud': False}})
                payload = jwt.decode(
                    token,
                    key,
                    **jwt_config
                )
...
    return decorated
```

**Comment by dmi3y** [comment]
I think we can go alone with some option for explicitly disabling the audience verification.

**Comment by mianelson** [comment]
Merged #4
